copter/source/docs/traditional-helicopter-tuning.rst

```diff
@@ -129,28 +64,21 @@ from the defaults.
 
 ** This param name changed in ArduPilot 4.0 and later.
 
+.. note:: before tuning, it is recommended that notch filtering (see :ref:`common-imu-notch-filtering`) be setup and that high frequency vibrations are not causing the "leans" (see :ref:`traditional-helicopter-tips`)
+
 Tuning the Yaw Axis (Rudder)
 ============================
+
 It is recommended to make sure the tail functions properly before proceeding
 with tuning pitch and roll.
 
 **Important Note** - UAV helicopters, as opposed to sport helicopters, will
 usually be running low headspeed and higher disc loading. With a mechanically
 driven tail this also means lower than normal tail speed and reduced tail
 authority. If your helicopter meets this description, it is recommended to set
-:ref:`ATC_RAT_YAW_VFF<ATC_RAT_YAW_VFF>` to 0.05 before the first test hover.
+:ref:`ATC_RAT_YAW_VFF<ATC_RAT_YAW_VFF>` to 0.024 before the first test hover.
```

Comment:
thought it was a typo since the table below says use 0.024.....but now I see that the note should be placed AFTER the table as an exception....will move below and keep the 0.05...

Reply:
yep that would be good.  Thanks.

Reply:
I am working off of your branch.  I fixed this.